sql/src/main/java/io/crate/planner/Planner.java

```diff
@@ -371,6 +371,10 @@ private UpsertById processInsertStatement(InsertFromValuesAnalyzedStatement anal
             List<String> partitions = analysis.generatePartitions();
             String[] indices = partitions.toArray(new String[partitions.size()]);
             for (int i = 0; i < indices.length; i++) {
+                if (analysis.tableInfo().isAClosedIndex(indices[i])) {
+                    LOGGER.warn(String.format(Locale.ENGLISH, "Trying to insert value into closed partition: %s", indices[i]));
```

Comment:
It would just not insert the data into the partition I believe - the use case for this was in case someone had closed a partition and had forgotten about it, prior to the inserts being done. If you think a check in the planner is inappropriate, where would you suggest the right point to check would be?